backend/src/documents/upload_document.py

```diff
@@ -0,0 +1,62 @@
+import base64
+import os
+import uuid
+
+from src import context
+from src.external.aws.s3 import S3
+from src.storage import CloudStorage, CloudStorageException
+
+appContext = context.ApplicationContext()
+appContext.register(CloudStorage, S3())
+
+
+def upload_file_data(body, bucket_name, default_folder) -> str:
+    filedata = generate_file_data(body)
+    upload_to_s3(filedata, bucket_name, default_folder)
+    return filedata.get("document_id")
+
+
+def generate_file_data(body):
+    if not all(k in body for k in ["file_content", "file_name"]):
+        raise ValueError("Missing required fields")
```

Comment:
I don't think we need this check going forward.

Reply:
K, let me take a closer look because I didn't re-evaluate this check specifically, I only moved it.